### PR TITLE
2020-05-22 IAC Team Blog Post

### PR DESCRIPTION
**Pull request by pmcmaw**

### CODE REVIEW

File: _posts/2020-05-22-status-update.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Thank you to [Victor Cabezas](https://github.com/Wiston999) for a fantastic PR for [puppetlabs/kubernetes](https://github.com/puppetlabs/puppetlabs-kubernetes). Its great to see such attention to detail in a PR from a community member.
+
+Thank you to [andeman](https://github.com/andeman)for contributing to our [puppetlabs/mysql](https://github.com/puppetlabs/puppetlabs-mysql) module and providing us with such a great feature.
+
+Thank you to [https://github.com/vchepkov](Vadym Chepkov) for your [Docker PR](https://github.com/puppetlabs/puppetlabs-docker/pull/613). Housekeeping on our modules is always beneficial and we really appreciate you taking the time to contribute.
+
+## PDKSync Runs
+
+ [David](https://github.com/david22swan) has ran 2 PDKSync runs this week to include generic template updates in our modules and to add Ubuntu 20.04 support and [Danny](https://github.com/carabasdaniel) has been running one to switch our default provisioner from vmpooler to abs.
```
Comment (sanfrancrisko) [inline _posts/2020-05-22-status-update.md:51]:
```suggestion
[David](https://github.com/david22swan) has performed 2 PDKSync runs this week to include generic template updates in our modules and to add Ubuntu 20.04 support. [Danny](https://github.com/carabasdaniel) has been running one to switch our default provisioner from vmpooler to abs.
```

File: _posts/2020-05-22-status-update.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  - mysql
+  - docker
+  - pdksync
+  - pdk
+
+---
+The IAC team has been working on a lot of new things over the past week. We want to share with our community some pretty interesting things. Here's a brief overview of what we've been up to:
+
+## Diana has been made permanent!
+We are absolutely delighted to announce our current intern Diana has recently been offered a permanent job role. Diana has been a major influence on our community reporting tool for when we are triaging PRs. We are so excited that she will be hanging around! We are all looking forward to have a celebratory beverage together at some point in the future.
```
Comment (carabasdaniel) [inline _posts/2020-05-22-status-update.md:29]:
```suggestion
## Daiana has been made permanent!
We are absolutely delighted to announce our current intern, Daiana, has recently been offered a permanent job role. Daiana has been a major influence on our community reporting tool for when we are triaging PRs and releasing modules. We are so excited that she will be hanging around! We are all looking forward to have a celebratory beverage together at some point in the future.
```

File: _posts/2020-05-22-status-update.md (hunk 1 above)
Comment (carabasdaniel) [inline _posts/2020-05-22-status-update.md:51]:
```suggestion
 [David](https://github.com/david22swan) has ran 2 PDKSync runs this week to include generic template updates in our modules and to add Ubuntu 20.04 support and [Danny](https://github.com/carabasdaniel) has been running one to switch our default provisioner from VMPooler to ABS for our weekly and pre-release CI runs. 
```

File: _posts/2020-05-22-status-update.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+[Ciaran](https://github.com/sanfrancrisko) now knows his neighborhood by the back of his hand, has taken up more cooking opportunities at home (he currently has enough Bolognese in his freezer to feed the whole IAC team), is getting a big nostalgia hit playing Streets of Rage 4 online with his mate and has big plans on getting back into playing guitar, it just hasn’t happened yet. Sounds like Ciaran has been doing great at keeping busy!
+
+[Danny](https://github.com/carabasdaniel) has found himself spending more time chilling out and reading books and when he is not doing that he is ‘fine tuning’ his video gaming skills.
+
+[David](https://github.com/david22swan) carries out a vigorous home workout regime on Tuesday, Wednesday and Thursdays. Way to go David, serious commitment there!
+
+[Sheena](https://github.com/sheenaajay) is currently home schooling 2 lovely girls. Although when they have some down time she is exploring new recipes with them.
+
+[Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy I wonder is this a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?
```
Comment (sanfrancrisko) [inline _posts/2020-05-22-status-update.md:73]:
```suggestion
[Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy. I wonder is this a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?
```

Comment (carabasdaniel) [inline _posts/2020-05-22-status-update.md:73]:
```suggestion
[Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy. I wonder if this is a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?
```

Comment (sanfrancrisko) [reply _posts/2020-05-22-status-update.md:73]: ...and one more `s/Diana/Daiana/` there too 😉

**Pull request merged**